### ISSUE TITLE
UI: Brown text over bagel is difficult to read

### ISSUE THREAD

**Issue by castavridis** [tudor-seserman/First_Python_Projects#4]
For accessibility purposes, it is crucial that a text's color has a sufficiently high contrast ratio compared to the background color(s) it is presented on. Higher contrast enables people with all levels of vision to read a website's content. 

Maintaining an accessible website is no longer a nice to have. Websites that do not meet specific accessibility standards can open up their parent company to litigation. There's a [landmark case](https://www.jdsupra.com/legalnews/court-finds-domino-s-pizza-violated-the-2182635/) in which Domino's was found in violation of the ADA's accessibility standards. In that case, Domino's did not have adequate screen-reader accessibility.

The best practice is to have your foreground text color pass both WCAG AA and WCAG AAA tests. You may check if the colors you've chosen pass by visiting: https://webaim.org/resources/contrastchecker/. There are other checkers as well.

See how your text stacked up: https://webaim.org/resources/contrastchecker/?fcolor=653104&bcolor=B06416
